lib/src/Base/Algo/CorrectedLeaveOneOut.cxx

```diff
@@ -78,7 +78,6 @@ Scalar CorrectedLeaveOneOut::run(LeastSquaresMethod & method,
   if (y.getDimension() != 1) throw InvalidArgumentException(HERE) << "Output sample should be unidimensional (dim=" << y.getDimension() << ").";
   if (y.getSize() != sampleSize) throw InvalidArgumentException(HERE) << "Samples should be equally sized (in=" << sampleSize << " out=" << y.getSize() << ").";
   const Scalar variance = y.computeVariance()[0];
-  if (!(variance > 0.0)) throw InvalidArgumentException(HERE) << "Null output sample variance.";
```

Comment:
A zero variance means a constant sample, which will mostly be learned with zero error for any sensible choice of functional basis. Instead of throwing an exception as if it was not possible for the algorithm to learn the meta-model IMO it is much more reasonable to forget about relative error.

Reply:
But you might encounter issues with `linear system solving` as lines are equal. How do you tackle this?

Reply:
In the case of SVD and QR methods it will return the solution of minimal norm, which is ok, and for CholeskyMethod it should throw an exception anyway.